src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const clonePosition = 1;
+const clonedPosition = 4;
+
+for (const row of table.rows) {
+  row.cells[clonedPosition].before(row.cells[clonePosition].cloneNode(true));
+}
```

Comment:
```suggestion
for (const row of document.querySelectorAll('tr')) {
  row.cells[4].before(row.cells[1].cloneNode(true));
}
```
Гениально) Ну а вот так можно в 3 строки

Reply:
```suggestion
for (const row of document.querySelectorAll('tr')) { row.cells[4].before(row.cells[1].cloneNode(true)); }
```
Даже в 1 😂